### PR TITLE
[0.3.x] CAL-392 Suppress 'Downloading/Downloaded' status lines in logs

### PR DESCRIPTION
**Pull request by mackncheesiest**
#### What does this PR do?
Suppresses lines of the form 
```
18:43:23 [INFO] Downloading: http://NEXUS/PATH/TO/ARTIFACT
18:43:23 [INFO] Downloaded: http://NEXUS/PATH/TO/ARTIFACT (27 KB at 985.4 KB/sec)
```
from the Jenkins Pipeline build logs

#### Who is reviewing it? 
@vinamartin @blen-desta @oconnormi @mcalcote @LinkMJB 

#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@bdeining
@clockard

#### How should this be tested? (List steps with links to updated documentation)
It has been verified both on a downstream project and in [CAL master](https://github.com/codice/alliance/commit/4f9e0c933d3dc541d88b838e567b521d7d878f72) that only the desired logs are suppressed.

#### Any background context you want to provide?
As a bonus, this also brings the 0.3.x pipeline up to date with master with regards to #477

#### What are the relevant tickets?
[CAL-392](https://codice.atlassian.net/browse/CAL-392)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by mackncheesiest** [comment]
build now (even though the Jenkinsfile doesn't apply to PR CI currently...)

**Comment by mackncheesiest** [comment]
build now

**Comment by mackncheesiest** [comment]
I don't think I'm a fan of that `Update branch` button

**Comment by mackncheesiest** [comment]
build now

File: Jenkinsfile [hunk 1]
```diff
@@ -179,78 +164,70 @@ pipeline {
...
-                                        }
+            parallel {
+                stage ('SonarCloud') {
+                    steps {
+                        withMaven(maven: 'M35', jdk: 'jdk8-latest', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${LARGE_MVN_OPTS} ${LINUX_MVN_RANDOM}') {
+                            withCredentials([string(credentialsId: 'SonarQubeGithubToken', variable: 'SONARQUBE_GITHUB_TOKEN'), string(credentialsId: 'sonarqube-token', variable: 'SONAR_TOKEN')]) {
+                                script {
+                                    // If this build is not a pull request, run sonar scan. otherwise run incremental scan
+                                    if (env.CHANGE_ID == null) {
+                                        sh 'mvn -q -B -Dcheckstyle.skip=true org.jacoco:jacoco-maven-plugin:prepare-agent install sonar:sonar -Dsonar.host.url=https://sonarcloud.io -Dsonar.login=$SONAR_TOKEN  -Dsonar.organization=codice -Dsonar.projectKey=org.codice.alliance -Dsonar.exclusions=${COVERAGE_EXCLUSIONS} -pl !$DOCS,!$ITESTS $DISABLE_DOWNLOAD_PROGRESS_OPTS'
```
Comment (clockard) [inline Jenkinsfile]: The project key for alliance should be `org.codice:alliance` and not `org.codice.alliance`

Comment (mackncheesiest) [reply Jenkinsfile]: Good catch. Looks like that's what's currently in place on master

**Comment by mackncheesiest** [comment]
build now

**Pull request merged**